### PR TITLE
Add OPA alternatives for policy as code

### PR DESCRIPTION
**Pull request by JonZeolla**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [ ] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [ ] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to websites have been formatted as [TEXT](URL)
- [ ] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #<insert number here>.

Thank you again for your contribution :smiley:

### CODE REVIEW

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md
Comment (JonZeolla) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:380]: This is the primary addition; other changes are cleanup

File: cheatsheets/Kubernetes_Security_Cheat_Sheet.md
Comment (JonZeolla) [inline cheatsheets/Kubernetes_Security_Cheat_Sheet.md:392]: Minor change here too

**Comment by szh** [comment]
The changes look good, but the commits are being flagged as Unverified.

**Comment by JonZeolla** [comment]
@szh that’s because GitHub doesn’t support gitsign. Do you need me to PGP sign instead?

**Comment by JonZeolla** [comment]
Also can someone run the linter so I can ensure it passes?

**Comment by szh** [comment]
> @szh that’s because GitHub doesn’t support gitsign. Do you need me to PGP sign instead?

GitHub is showing the warning because you have [Vigilant Mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits) enabled, which tells GitHub to flag any commits that aren't signed. If you don't normally sign commits with PGP, do you mind disabling vigilant mode so it doesn't flag these commits? Otherwise it would be helpful if you could PGP sign them.

**Comment by szh** [comment]
> Also can someone run the linter so I can ensure it passes?

I don't have write access - @mackowski can you please run the GitHub ACtions?

**Comment by JonZeolla** [comment]
@szh squashed the commits into one and GPG signed it, we have a calming green Verified now :)

**Review by szh — APPROVED** [review]
Awesome, LGTM! I think the maintainers might be taking time off for the holidays, so it might be a few days before this gets merged.

**Review by mackowski — APPROVED** [review]
Looks good

**Comment by mackowski** [comment]
Thank you @JonZeolla and @szh!

**Comment by JonZeolla** [comment]
Did a quick sanity check of the rendered webpage and it LGTM

https://cheatsheetseries.owasp.org/cheatsheets/Kubernetes_Security_Cheat_Sheet.html

**Pull request merged**
